.github/workflows/update-browser-version.yml

```diff
@@ -24,15 +24,10 @@ jobs:
       - name: Set up Node.js
         uses: actions/setup-node@5e21ff4d9bc1a8cf6de233a3057d20ec6b3fb69d # v3.8.1
         with:
-          node-version: 18
+          node-version: lts/*
```

Comment:
Could we leave the version unchanged in this PR?

Ideally we should use the same version in _every_ workflow. It's currently inconsistent: https://github.com/search?q=repo%3AGoogleChromeLabs%2Fchromium-bidi%20node-version&type=code

It's better to address that in a separate PR

Reply:
https://github.com/GoogleChromeLabs/chromium-bidi/pull/1315